extension/src/experiments/webview/messages.ts

```diff
@@ -136,6 +136,9 @@ export class WebviewMessages {
           this.showPlots()
         ])
 
+      case MessageFromWebviewType.UPDATE_HEADER_DEPTH_SETTINGS:
+        return commands.executeCommand('workbench.action.openWorkspaceSettings')
```

Comment:
The closest thing I could find to linking to the dvc config settings was to just link to workspace settings in general... 

The only doable solution I can think of is creating a command that allows the user to update the config from the command palette 🤔

Reply:
All you need to do is use [setConfigValue](https://github.com/iterative/vscode-dvc/blob/1efac498e4589de18877530c7210290de51a0254/extension/src/vscode/config.ts#L19). You can set the value through any means you want.

Reply:
You will need to use a `quickInput` to get the required number.